pkgs/top-level/all-packages.nix

```diff
@@ -15867,6 +15867,8 @@ with pkgs;
 
   ograc = callPackage ../development/tools/rust/ograc { };
 
+  opensmalltalk-vm = callPackage ../development/compilers/opensmalltalk-vm { };
```

Comment:
This isn't good because `nix build -f nixpkgs opensmalltalk-vm` builds something with a binary called `squeak` that doesn't run squeak. I think the VMs could be moved into the top-level packages or otherwise replace the current `squeak` package.

```
  inherit (callPackages ../development/compilers/opensmalltalk-vm { })
    newspeak-cog-spur squeak-cog-spur;
```

Reply:
I see what you mean: I'm a little confused about this, because I usually use flakes which gives an error if you try and build the whole attrset. It also gives tab completion off of `opensmalltalk-vm` which is nice.
```
$ nix build ~/nixpkgs#opensmalltalk-vm
error: flake output attribute 'legacyPackages.aarch64-linux.opensmalltalk-vm' is not a derivation

$ nix build ~/nixpkgs#opensmalltalk-vm.<TAB>
~/nixpkgs#opensmalltalk-vm.override
~/nixpkgs#opensmalltalk-vm.overrideDerivation
~/nixpkgs#opensmalltalk-vm.squeak-cog-spur
~/nixpkgs#opensmalltalk-vm.squeak-stack-spur
```
Do you happen to know how `nix build -f` chooses which element of the attrset to build? Also, I think this should be less of an issue now that the newspeak VM has a binary called `newspeak` and not `squeak`.

Reply:
I'm not sure what `nix build -f` is doing but I think in general we want to flatten what is in `all-packages` except for the big language library attrsets.

Reply:
I suppose it doesn't matter if we alias `squeak` into this attrset.